golang/static/index.html

```diff
@@ -56,8 +56,8 @@
         })
 
         let isHTTPS = location.protocol !== 'https:'
-        signals.connect((isHTTPS ? "ws" : "wss") + "://" + location.host + "/room")
-
+        signals.connect((isHTTPS ? "ws" : "wss") + "://localhost:3001/room")
+        console.log(location.host)
```

Comment:
These are debugging right? This keeps the example working on `https://club.pedanticorderliness.com`